l10n_br_base/models/res_partner.py

```diff
@@ -133,15 +133,17 @@ def _check_cnpj_inscr_est(self):
     def _check_cnpj_cpf(self):
         result = True
         for record in self:
-            country_code = record.country_id.code or ""
-            if record.cnpj_cpf and country_code.upper() == "BR":
-                if record.is_company:
-                    if not cnpj_cpf.validar(record.cnpj_cpf):
-                        result = False
-                        document = "CNPJ"
-                elif not cnpj_cpf.validar(record.cnpj_cpf):
-                    result = False
-                    document = "CPF"
+            if record.country_id:
+                country_code = record.country_id.code or ""
```

Comment:
nao eh algo critico, mas esse ultimo `or ""` eh inutil ja que nao vai fazer nada com o country_code caso ele for vazio depois.